src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,43 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Opening Records in Modal Windows
+
+Record lists have an option to open records in a modal on record click.
```

Comment:
`in a modal on record click` -> `in a modal window`

The "on record click" bit sounds off to me